hbase-client/src/main/java/org/apache/omid/transaction/SnapshotFilterImpl.java

```diff
@@ -395,8 +427,8 @@ private Get createPendingGet(Cell cell, int versionCount) throws IOException {
                     }
                 }
 
-                if (isCellInTransaction(cell, transaction, commitCache) ||
-                    isCellInSnapshot(cell, transaction, commitCache)) {
+                if (isCellInTransaction(cell, transaction, commitCache).isPresent() ||
+                    isCellInSnapshot(cell, transaction, commitCache).isPresent()) {
```

Comment:
Isnt isCellInTransaction  a subgroup of isCellInSnapshot ? why call both?

Reply:
Same as before